test/extension/loadable_extension_demo.cpp

```diff
@@ -11,6 +14,88 @@ inline string_t hello_fun(string_t what) {
 	return "Hello, " + what.GetString();
 }
 
+DUCKDB_API inline void TestAliasHello(DataChunk &args, ExpressionState &state, Vector &result) {
```

Comment:
The windows build is breaking because of the added `DUCKDB_API` here. This should only be added inside the actual DuckDB library.